contrib/msi/podman.wxs

```diff
@@ -24,16 +24,15 @@
               <CreateFolder/>
             </Component>
             <Component Id="MainExecutable" Guid="73752F94-6589-4C7B-ABED-39D655A19714">
-              <File Id="520C6E17-77A2-4F41-9611-30FA763A0702" Name="podman-remote-windows.exe" Source="bin/podman-remote-windows.exe"/>
-              <File Id="A14218A0-4180-44AC-B109-7C63B3099DCA" Name="podman.bat" Source="podman.bat" KeyPath="yes"/>
+              <File Id="520C6E17-77A2-4F41-9611-30FA763A0702" Name="podman.exe" Source="bin/podman-remote-windows.exe" KeyPath="yes"/>
             </Component>
           </Directory>
         </Directory>
       </Directory>
     </Directory>
 
     <Property Id="setx" Value="setx.exe"/>
-    <CustomAction Id="ChangePath" ExeCommand="PATH &quot;%PATH%;[INSTALLDIR] &quot;" Property="setx" Execute="deferred" Impersonate="yes" Return="check"/>
+    <CustomAction Id="ChangePath" ExeCommand="PATH &quot;%PATH%;[INSTALLDIR]&quot;" Property="setx" Execute="deferred" Impersonate="yes" Return="check"/>
```

Comment:
@baude I think using setx isn't a good option. It doesn't remove the entry on uninstall.

I stumbled across this SO post: https://stackoverflow.com/questions/1931586/can-anyone-give-me-a-example-of-modifying-windows-environment-system-variables-i
wix toolset doc: https://wixtoolset.org/documentation/manual/v3/xsd/wix/environment.html

So i think the correct way would be to use this line: `<Environment Id="PATH" Name="PATH" Value="[INSTALLDIR]" Permanent="no" Part="last" Action="set" System="yes" />` and I think it has to be placed between line 27 and 28.

Reply:
@Luap99 thanks! I'll take a look.  Many, many, thanks

Reply:
@jwhonce this is starting to smell like a difference between wixl and wix?

Reply:
@baude Unfortunately the environment key is not supported by wixl. I found this PR: https://gitlab.gnome.org/GNOME/msitools/-/merge_requests/11

Reply:
@Luap99 Yes, would love to do this the "right" way once wixl supports it. /cc @baude